src/shared/components/react-live/live-editor.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { LiveProvider, Editor } from 'react-live'
+
+export default class LiveEditor extends Component {
+  static contextTypes = LiveProvider.childContextTypes
+
+  constructor (props, context) {
+    super(props, context)
```

Comment:
Remove the `props` and `context`. You don't need to explicitly pass them unless you use them in the constructor.